common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -28,7 +28,9 @@ try {
                 var adStyles = window.getComputedStyle(ad);
 
                 // Due to an update to AdblockPlus we now need an extra check for adblock being used on Firefox as the sacrificial ad check does not work.
-                adBlockers.active = adStyles.getPropertyValue('display') === 'none'|| adStyles.getPropertyValue('-moz-binding').indexOf('about:')!== -1;
+                var displayProp = adStyles.getPropertyValue('display');
+                var mozBindingProp = adStyles.getPropertyValue('-moz-binding');
+                adBlockers.active = displayProp === 'none' || (mozBindingProp !== null && mozBindingProp.indexOf('about:') !== -1);
```

Comment:
I think that'd definitely work. Not sure if we have a preferred style though? There's !x, x != null, typeof(x) !== 'null' etc